dipy/reconst/ivim.py

```diff
@@ -10,6 +10,11 @@
 cvxpy, have_cvxpy, _ = optional_package("cvxpy")
 
 
+# global variables for bounding least_squares in both models
+BOUNDS_TRF = ([0., 0., 0., 0.], [np.inf, .2, 1., 1.])
+BOUNDS_VP = ([0.01, 0.005, 10**-4], [0.3, 0.02,  0.003])
```

Comment:
pep8: Can you remove one space before `0.003`

Reply:
> Overall it looks good. Just take care of you rebase because you are deleting previous work like warning management and `setup_module` for pytest.
> 
> Thank you @ShreyasFadnavis for this update on IVIM

On it!

Reply:
@skoudoro ! Thank you 👍 I have made the suggested changes...